README.md

```diff
@@ -1,110 +1,157 @@
-Make sure that you have set up ssh for gitlab (https://docs.gitlab.com/ee/ssh/)
-Make sure you have `kustomize` version at least `3`
-You can then start your environment using
+install `pre-commit` https://pre-commit.com/
 
 ```bash
-minikube start
-tilt up
+$ curl https://pre-commit.com/install-local.py | python -
 ```
 
-#### Without Docker (Only inspirehep Without The UI)
+And run
 
 ```bash
-$ ./scripts/bootstrap
-$ ./docker-inspire up -d
-$ ./scripts/setup
-$ ./scripts/server
-$ firefox localhost:5000
+$ pre-commit install
 ```
 
-#### Without Docker Only The UI
+### nginx
+
+#### macOS:
 
 ```bash
-$ cd ui/
-$ yarn start
-$ firefox localhost:3000
+$ brew install nginx
 ```
 
-You can proxy a server by changing the `ui/package.json`
+#### Linux (Ubuntu):
 
-```json
-{
-  ...
-  "proxy": "http://A_PROXY_SERVER",
-  ...
+```bash
+$ apt-get update
+$ apt-get install nginx
+```
+
+Add the following configuration:
+
+```nginx
+server {
+  listen 8080 default_server;
+  server_name  _;
+
+  location / {
+    proxy_pass  http://localhost:3000/api;
+    proxy_set_header Host localhost:3000;
+    proxy_http_version 1.1;
+  }
+
+  location /api {
+    proxy_pass         http://localhost:8000/api;
+    proxy_set_header Host localhost:8000;
+    proxy_http_version 1.1;
+  }
 }
 ```
 
-### Import records
+* On macOS `/usr/local/etc/nginx/nginx.conf`
+* On Linux `/etc/nginx/conf.d/default.conf`
 
-There is a command `inspirehep importer records` which accepts url `-u`, a directory of `JSON` files `-d` and `JSON` files `-f`.
-A selection of demo records can be found in `data` directory and they are structure based on the record type (i.e. `literature`). Examples:
+And reload `nginx`
 
-#### With url
+### Docker & Docker Compose
+
+Follow the guide https://docs.docker.com/compose/install/
+
+---
+
+## Installation
+
+### Backend
 
 ```bash
-# Local
-$ inspirehep importer records -u https://labs.inspirehep.net/api/literature/20 -u https://labs.inspirehep.net/api/literature/1726642
-# Docker
-$ ./docker-inspire run --rm web poetry run inspirehep importer records -u https://labs.inspirehep.net/api/literature/20 -u https://labs.inspirehep.net/api/literature/1726642
+$ cd backend
+$ poetry install
 ```
 
-#### With directory
+### UI
 
 ```bash
-# Local
-$ inspirehep importer records -d data/literature
-# Docker
-$ ./docker-inspire run --rm web poetry run inspirehep importer records -d data/literature
+$ cd ui
+$ yarn install
 ```
 
-#### With files
+---
+
+## Setup
+
+First you need to start all the services (postgreSQL, Redis, ElasticSearch, RabbitMQ)
 
 ```bash
-# Local
-$ inspirehep importer records -f data/literature/374836.json -f data/authors/999108.json
-# Docker
-$ ./docker-inspire run --rm web poetry run inspirehep importer records -f data/literature/374836.json -f data/authors/999108.json
+$ docker-compose -f docker-compose.travis.yml up -d
+```
+
+And initialize database and ES
+
+```bash
+$ ./backend/scripts/setup
```

Comment:
This won't work. You have to be in `backend` directory to run ./scripts/setup otherwise poetry won't be able to locate pyproject.yaml